src/WebJobs.Script.WebHost/FunctionRequestInvoker.cs

```diff
@@ -120,10 +122,29 @@ private static bool IsHomepageDisabled
         {
             var authorizationResult = await AuthorizationLevelAttribute.GetAuthorizationResultAsync(request, _secretManager, functionName: _function.Name);
             var authorizationLevel = authorizationResult.AuthorizationLevel;
+            if (authorizationLevel > AuthorizationLevel.Anonymous && authorizationLevel < AuthorizationLevel.User)
```

Comment:
What is this if-condition checking for? Can we make the intention more explicit? Even something like this would make it significantly more readable:
```csharp
bool isApiKeyAuth = authorizationLevel > AuthorizationLevel.Anonymous && authorizationLevel < AuthorizationLevel.User;
if (isApiKeyAuth)
{
    // ...
}
```

Reply:
Checking that it is an authkey is exactly what it is doing. I'll make this more clear.

---
In reply to: [176887060](https://github.com/Azure/azure-functions-host/pull/2577#discussion_r176887060) [](ancestors = 176887060)